app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMain.java

```diff
@@ -138,8 +164,14 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
             }
 
         });
-
         return view;
+    }
+
+    private void setText (String text) {
```

Comment:
Is there any specific reason for creating a function ```setText``` instead of using ```tv.setText```?
Same for ```setAdapter``` method below.